radicals/urls.py

```diff
@@ -21,17 +21,18 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', lambda request: HttpResponseRedirect('/home_page/')),
-    path('customers/', include('back_office.customers.urls')),
-    path('invoices/', include('back_office.invoices.urls')),
-    path('orders/', include('back_office.orders.urls')),
-    path('products/', include('back_office.products.urls')),
-    path('sales/', include('back_office.sales.urls')),
-    path('settings/', include('back_office.settings.urls')),
-    path('about/', include('front_office.about_page.urls')),
-    path('account/', include('front_office.account_page.urls')),
-    path('home/', include('front_office.home_page.urls')),
-    path('products/', include('front_office.products_page.urls')),
-    path('services/', include('front_office.services_page.urls')),
-    path('shopping_cart/', include('front_office.shopping_cart_page.urls')),
+    path('', lambda request: HttpResponseRedirect('frontoffice/home/')),
+    path('backoffice/customers/', include('back_office.customers.urls')),
+    path('backoffice/invoices/', include('back_office.invoices.urls')),
+    path('backoffice/orders/', include('back_office.orders.urls')),
+    path('backoffice/products/', include('back_office.products.urls')),
+    path('backoffice/sales/', include('back_office.sales.urls')),
+    path('backoffice/settings/', include('back_office.settings.urls')),
+    path('frontoffice/about/', include('front_office.about_page.urls')),
```

Comment:
Ticket only requests to change backoffice, front office should stay with simple URLs